spec/commands/open_spec.rb

```diff
@@ -2,11 +2,13 @@
 require "spec_helper"
 
 describe "bundle open" do
-  before :each do
-    install_gemfile <<-G
-      source "file://#{gem_repo1}"
-      gem "rails"
-    G
+  unless example.metadata[:skip_before]
+    before :each do
+      install_gemfile <<-G
+        source "file://#{gem_repo1}"
+        gem "rails"
+      G
+    end
```

Comment:
The [test output in Travis](https://travis-ci.org/bundler/bundler/jobs/154594024#L399) seems to install _this_ Gemfile.

The `skip_before: true` metadata is ignored.

How about:

- `before(:each) do |example|` to have access to the `example` inside the `before`
- add a line inside the block: `next if example.metadata[:skip_before]`
- remove the wrapping `unless`/`end`